docs/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,801 @@
+    "\n",
+    "We will send a different type of evidence - a table, which must be a pandas DataFrame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6861fda-a0b4-49b4-8f53-c8f187da0c26",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "table = pd.DataFrame({'arbitrary_data': [3,4,5]})\n",
+    "table.name = 'my_table'\n",
+    "adapter.table_to_governance(table, source=source, overwrite_governance=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9da37aa7-5305-4ec9-8c49-a11d32116f16",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gov.get_evidence()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e4364e8-d5da-436e-a0b3-e38a1b283f11",
+   "metadata": {},
+   "source": [
+    "#### 3. Export the evidence to the platform.\n",
+    "\n",
+    "Exporting is straight forward! You can either export directly to the Platform or to a file. At the time of export, the uploaded evidence will be checked against the governance requirements specified on Platform and let you know what's missing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bddb93de-6139-4f22-9df0-b97019157541",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# export to API\n",
+    "gov.export()\n",
+    "\n",
+    "# export to file\n",
+    "gov.export(\"assessment_export.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6c71569-6152-42d6-aadc-da60a25eca52",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Other ways to label your evidence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21ea5e8a-2cb1-41ec-aafb-a10299774725",
+   "metadata": {},
+   "source": [
+    "On the Credo AI Platform, certain governance requirements may only apply to models that are tagged in a certain way. You can get the requirements tags from governance and apply it by passing a dictionary to the `model_tags` of the `Adapter`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd269b76-3fb7-4288-8b0a-bf16772b5b6a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gov.get_requirement_tags()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "861cdace-f849-4248-bdd7-cd5a86888902",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adapter = Adapter(governance = gov, model_name='My Model', model_tags={'model_type': 'regression'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee605487-1de4-4fe7-8df2-cb8947a260cd",
+   "metadata": {},
+   "source": [
+    "If you need to label your evidence with more information than just `source` you can do that too."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0797a51",
+   "metadata": {},
+   "source": [
+    "## Lens in 5 minutes"
```

Comment:
Do we need a lens quickstart in here? We could keep it simple and reference the proper Lens quickstart